acceptance/setup/pre_suite/15_setup_repos.rb

```diff
@@ -1,6 +1,14 @@
 def initialize_repo_on_host(host, os, nightly)
   case os
   when :debian
+
+    # For openjdk8
+    if host['platform'].version == '8'
+      create_remote_file(host,
+                         "/etc/apt/sources.list.d/jessie-backports.list",
+                         "deb http://httpredir.debian.org/debian jessie-backports main")
```

Comment:
Yeah, I think I'd recommend httpredir these days, but I don't know that it's been used heavily inside our CI infrastructure yet, so we might want to wait a bit and see how this goes first.